### DISCUSSION TITLE
Basic set up

### DISCUSSION THREAD

**Discussion by schneider82** [gravitational/teleport#9810] [General]
There seems to be a lot of expert help but I can't seem to find the basics.

The pop up gives a set-up that only allows the active user and I can't see how that is changed... it is also temporary but when I use the demo environment it had a script that ran and connected the agent so it was accessible. Why doesn't the standard package have this? is there a page I'm missing? I have tried the documentation but there doesn't seem to be a Teleport for dummies... also Teleport searches in google are pretty useless given the common word. The program looks amazing if only I could get it to work the first time and I would be off and running.

**Comment by benarent** [comment]
Hi @schneider82, I would recommend starting with the Teleport Lab https://goteleport.com/docs/getting-started/docker-compose/ as a good way to get started. 

Can you also expand on. Which script? 

> it is also temporary but when I use the demo environment it had a script that ran and connected the agent so it was accessible. Why doesn't the standard package have this? is there a page I'm missing?
